### DISCUSSION TITLE
iOS background copy

### DISCUSSION THREAD

**Discussion by owerrride** [immich-app/immich#2584] [General]
Can you tell me if this works on ios? I have enabled both toggle switches however it does not work. iPhone 12 Pro Max, ios 16.4.1
Thanks in advance

**Comment by alextran1502** [comment]
Have you enabled "Background App Refresh" in your phone setting? The background task will be queued by the iOS scheduler and will run when it is allowed to run, this can varied from every 10 to 15 minutes interval depends on your usage frequency of the app. Also the app must be in the background and not killed (swipe up)
